applications/openshift/integrity/cluster_version_operator_exists/rule.yml

```diff
@@ -0,0 +1,47 @@
+prodtype: ocp4
+
+title: Ensure that Cluster Version Operator verify integrity
```

Comment:
s/verify/verifies/

Reply:
I think instead of this, what we're checking is that it's available. Let's reflect that in the title